### PR TITLE
Issue 4197:  Fix Controller REST api Documentation.

### LINKED ISSUE #4197: Update REST api documentation for the Pravega controller
**Problem description**
The current REST API documentation for Pravega controller is not upto date.

**Problem location**
restapis.md

**Suggestions for an improvement**
- Ensure the latest `swagger2markup-cli-1.3.3.jar` is used to generate the documentation.
- Ensure the REST API documentation is upto date.

### PR DESCRIPTION
**Pull request by shrids**
**Change log description**  
Ensure the autogenerated REST Api documentation is updated

**Purpose of the change**  
Fixes #4197 

**What the code does**  
- Ensure the documentation reflects the latest code. e.g: ScaleMetadata information.
- Update the swagger2markup-cli jar version to 1.3.3.

**How to verify it**  
No changes to functionality.

### CODE REVIEW

File: documentation/src/docs/rest/restapis.md [hunk 1]
```diff
@@ -88,7 +88,6 @@ Create a new scope
 
 ##### Request body
 ```json
-json :
```
Comment (shiveshr) [inline documentation/src/docs/rest/restapis.md]: why are we removing the json element?

File: documentation/src/docs/rest/restapis.md [hunk 2]
```diff
@@ -489,20 +481,10 @@ List streams within the given scope
 ```
 
 
-##### Request query
```
Comment (shiveshr) [inline documentation/src/docs/rest/restapis.md]: why is this section removed?

File: documentation/src/docs/rest/restapis.md (hunk 1 above)
Comment (shrids) [reply documentation/src/docs/rest/restapis.md]: This is an auto-generated document from the swagger yaml file. This does not cause any changes to formatting. The complete document can be seen @ https://github.com/shrids/pravega/blob/issue-fixRESTdocumentation/documentation/src/docs/rest/restapis.md

File: documentation/src/docs/rest/restapis.md (hunk 2 above)
Comment (shrids) [reply documentation/src/docs/rest/restapis.md]: This was auto-generated by the swagger2markup-cli-1.3.3.jar.  Also, we can observe that the query parameter for the REST api has been documented @ https://github.com/shrids/pravega/blob/issue-fixRESTdocumentation/documentation/src/docs/rest/restapis.md#parameters-6

File: documentation/src/docs/rest/restapis.md (hunk 1 above)
Comment (shiveshr) [reply documentation/src/docs/rest/restapis.md]: So why are we checking in a generated document?

**Pull request merged**